pkg/noderesourcetopology/README.md

```diff
@@ -98,6 +98,24 @@ profiles:
       cacheResyncPeriodSeconds: 5
 ```
 
+#### ScoringStrategy
+
+Topology-aware scheduler supports four scoring strategies. You can set a strategy via SchedulerConfigConfiguration, by setting `scoringStrategy` option. There are four supported strategies:
+
+* MostAllocated
+* BalancedAllocation
+* LeastAllocated
+* LeastNUMANodes
+
+MostAllocated, BalancedAllocation and LeastAllocated strategies work only for `single-numa-node` Topology Manager policy and indicate how score of the worker node will be calculated based
+on current utilization:
+
+* MostAllocated - favors node with the least amount of available resources
+* BalancedAllocation - favors node with balanced resource usage rate
+* LeastAllocated - favors node with the most amount of available resource
+
+**LeastNUMANodes** strategy works for all Topology Manager policies and favors nodes which require least amount of topology zones to satisfy resource requests for given pod.
```

Comment:
"The LeastNUMANodes strategy works with all the Topology Manager policies and favors nodes which require the least amount of topology zones to satisfy the resource requests for a given pod."

Reply:
👍